### ISSUE TITLE
Adding discussion on Jupyter Book 

### ISSUE THREAD

**Issue by lheagy** [jupyter4edu/jupyter-edu-book#84]
Jupyter-Book is a resource that we should mention! There are two potential place, it could fit in [Chapter 5](https://github.com/jupyter4edu/jupyter-edu-book/blob/master/05-jupyter-ecosystem.md#resources-for-authoring-jupyter-notebooks) or [Chapter 6](https://github.com/jupyter4edu/jupyter-edu-book/blob/master/06-getting-your-class-going-with-jupyter.md#distribution-and-collection-of-materials).
 
The resource in Chapter 5 are more about tools for authoring notebooks, so I vote that it be included in [Chapter 6](https://github.com/jupyter4edu/jupyter-edu-book/blob/master/06-getting-your-class-going-with-jupyter.md#distribution-and-collection-of-materials) where we discuss the distribution of notebooks. Thoughts?

cc @choldgraf

**Comment by choldgraf** [comment]
agreed - the goal of jupyter-book is to help people distribute their collections of notebooks+markdown files in a book-like form, so distribution seems like a good section for this

**Comment by arm61** [comment]
I was thinking of working on this next week (I have familiarity with the jupyter-book framework). If there is any particular point that should be driven home, limme know.

**Comment by choldgraf** [comment]
(lemme know if I can help helpful in this one!)

**Comment by arm61** [comment]
@choldgraf I was thinking that I would give it a bash and get some prose down then I'll refer to you to put in the facts (that I will assume you know better than I!) as necessary

**Comment by choldgraf** [comment]
sounds good, feel free to ping me :-) just

**Comment by arm61** [comment]
Sorry about the delay on this but last week and this week have been distracted by referees comments on papers. I will try and swing back to this before the month is out
